### PR TITLE
Add hue-ambiance plugin

### PR DESCRIPTION
**Pull request by Jallah123**
This PR adds the hue-ambiance plugin.

The plugin will connect with Philips Hue lights and change them based on in-game events.

Currently supported features:

- Skybox
- Level up notifier
- Low hp notifier
- Low prayer notifier
- Drop notifier
  - Drops above X price
  - COX drops
  - CG drops
- Custom zulrah colors

### CODE REVIEW

File: package/target_init.gradle [hunk 1]
```diff
@@ -54,6 +54,7 @@ allprojects {
 	def buildDir = new File(System.getenv("runelite.pluginhub.package.buildDir"));
 
 	task runelitePluginHubPackage(type: Jar) {
+        duplicatesStrategy = 'warn'
```
Comment (LlemonDuck) [inline package/target_init.gradle:57]: Why is this included?

Comment (Jallah123) [reply package/target_init.gradle:57]: Basically when creating a fat jar it copies all the files into a single jar. Whenever there are duplicates like LICENSE or NOTICE it will break.

Now it will keep both duplicates and log a warning that it happened. This way there is no loss of files and the build will succeed.

**Comment by Jallah123** [comment]
@LlemonDuck any more questions or do you think it is good to go?

File: package/target_init.gradle (hunk 1 above)
Comment (Nightfirecat) [reply package/target_init.gradle:57]: Why is this put here (which would affect all plugins) instead of in the build script for your plugin?

Comment (Jallah123) [reply package/target_init.gradle]: ah thanks didn't know that was possible. fixed it in my branch, will clean up the git history when I'm home :)

**Comment by Jallah123** [comment]
@Nightfirecat any other comments?
